apps/astarte_housekeeping/lib/astarte_housekeeping/utils/astarte_instance_id_type.ex

```diff
@@ -0,0 +1,17 @@
+defmodule AstarteInstanceIdType do
+  use Skogsra.Type
+
+  @impl Skogsra.Type
+  def cast(value)
+
+  def cast(value) when is_binary(value) do
```

Comment:
I've seen doc of Skogsra a bit, but seems there are no best-pratice to raise inside cast (otherwise, it should have implemented `cast!`

I guess instance id is UTF8, so the best way to solve could be:

```elixir
def cast(value) when is_binary(value) and byte_size(value) <= 41 do
{:ok, value}
end
```